static/js/containers/AutomaticEmailPage_test.js

```diff
@@ -93,18 +97,9 @@ describe('AutomaticEmailPage', () => {
   });
 
   it('should let you save an email', () => {
-    helper.fetchJSONWithCSRFStub.withArgs(
-      `/api/v0/mail/automatic_email/${GET_AUTOMATIC_EMAILS_RESPONSE[0].id}/`,
-      {
-        method: 'PATCH',
-        body: JSON.stringify({
-          email_subject: GET_AUTOMATIC_EMAILS_RESPONSE[0].email_subject,
-          email_body: GET_AUTOMATIC_EMAILS_RESPONSE[0].email_body,
-          sendAutomaticEmails: true,
```

Comment:
I'm just using `fetchMock` to mock the correct response, I can add an assertion to check that the request is formed correctly